src/Frontend.hs

```diff
@@ -172,15 +172,15 @@ type AulaMain =
 
 aulaMain :: ActionM m => ServerT AulaMain m
 aulaMain =
-       runHandler Page.viewRooms
+       forceChangeInitialPassword (runHandler Page.viewRooms)
```

Comment:
We could save as a session state if we rendered redirected once in the session on case of initial password.